common/src/main/java/com/sequenceiq/cloudbreak/logger/RestLoggerField.java

```diff
@@ -0,0 +1,25 @@
+package com.sequenceiq.cloudbreak.logger;
+
+public enum RestLoggerField {
```

Comment:
Do we need it as a separate enum?

Reply:
I thought it is nicer this way. Should I remove it ?